# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/generated-mochitest.ini
# Commit: c333871e1c59
# Full Hash: c333871e1c59abe6daf30c396e5e0a09591ac440
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2022-01-19 09:34:35
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1736793 - Add SurfaceTexture support to GLBlitHelper::BlitSdToFramebuffer() r=jgilbert,gfx-reviewers
#   
#   Blit support of android SurfaceTexture was removed by Bug 1640607.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D128992
# ==============================================================================

diff -r bb40a646b246 -r c333871e1c59 dom/canvas/test/webgl-conf/generated-mochitest.ini
--- a/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 19 02:03:31 2022 +0000
+++ b/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 19 02:13:48 2022 +0000
@@ -7002,6 +7002,7 @@
 subsuite = webgl2-core
 [generated/test_2_conformance2__textures__misc__npot-video-sizing.html]
 subsuite = webgl2-core
+fail-if = (os == 'android')
 [generated/test_2_conformance2__textures__misc__origin-clean-conformance-offscreencanvas.html]
 subsuite = webgl2-core
 skip-if = 1